templates/store/reducers/index.ts

```diff
@@ -0,0 +1,40 @@
+import { {{capitalize name}}Actions, {{capitalize name}}ActionTypes } from '../actions';
+
+export interface {{capitalize name}}State {
```

Comment:
I see that {{capitalize name}} is repeated mutiple times. Cant remeber how to do that but maybe we can define proper variables at the begginig of template and then reuse. Is thsi hanelbars template or something else? 
Im thinking of something like: 

{{setVar "cName" "capitalize name"}}

We will have short single variable which can be reused

Reply:
In handlebars there is no such thing as declaring variables out of the box. It's only possible through register custom handlebar helper, like e.g.:
```
handlebars.registerHelper("setVar", function(varName, varValue, options) {
  options.data.root[varName] = varValue;
});
```
In this case it is even more complicated, because it uses handlebar helper 'capitalize'.

Reply:
Added setVar helper and used for create common variable in this file.